src/main/java/com/flansmod/common/driveables/DriveableType.java

```diff
@@ -134,6 +136,9 @@
 	public int startSoundLength;
 	public String engineSound;
 	public int engineSoundLength;
+	
+	/**Track animation frames */
+	public int animFrames = 2;
```

Comment:
So if I load up an old tank it will be doing a 3 frame animation in the background, but they don't have any model pieces, so you'll see nothing? That seems okay